### PR TITLE
doc: add minutes for meeting Dec 6 2017

### PR DESCRIPTION
**Pull request by mhdawson**

### CODE REVIEW

File: meetings/2017-12-06.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+  * opinion from TSC in issue has mostly been -> not an option
+  * no opposition from those in meeting, will remove from agenda and say 'not an option' in 
+    issue.
+
+* events: add off alias to removeListener [#17156](https://github.com/nodejs/node/pull/17156)
+  * Matteo mentioned he was opposed
+  * Rod not a fan, but people seem to really want it
+  * Opposed to be it being semver minor.  Believe it is a bad idea but more ok if it is
+    semver minor.  
+  * Trevor ran into issues in past when we added an _property.  Cannot add to event
```
Comment (maclover7) [inline meetings/2017-12-06.md]: needs end `_` to close the italics

File: meetings/2017-12-06.md [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+  * phased approach to deprecate some parts earlier
+  * question to TSC is that an option ?
+  * opinion from TSC in issue has mostly been -> not an option
+  * no opposition from those in meeting, will remove from agenda and say 'not an option' in 
+    issue.
+
+* events: add off alias to removeListener [#17156](https://github.com/nodejs/node/pull/17156)
+  * Matteo mentioned he was opposed
+  * Rod not a fan, but people seem to really want it
+  * Opposed to be it being semver minor.  Believe it is a bad idea but more ok if it is
```
Comment (maclover7) [inline meetings/2017-12-06.md]: should the second `semver minor` in the sentence be `semver major`?

File: meetings/2017-12-06.md [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+  * Myles anyway we can land, with caveat not to backport until enough usage
+  * Matteo would like it never to go back
+  * Matteo is ok with 'do not land on 8'.
+  * James prefer semver major, land in 10. Matteo that would be even better.
+  * Jeremiah not in agreement.
+  * All in agreement that if it lands it should be semver major.
+  * Myles will put summary comment into issue.
+
+* Add remark dependencies  [#17320](https://github.com/nodejs/node/pull/17320)
+  * Added my Myles
```
Comment (maclover7) [inline meetings/2017-12-06.md]: my --> by

File: meetings/2017-12-06.md [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+  * When remark was added we decided not to add the dependencies, opposition was around
+    git performance.
+  * Myles did more recent investigation found some info that it might not be a perf issue
+  * Rich, maybe we should have invited @refack to to participate in the discussion.
+  * 1800 files is about 4% 
+  * Michael do we have the comparison between git clone time with/without
+  * Jeremiah, put into tarball, pull tarball and uncompress.  Myles maybe we should
+    do for eslint dependencies as well.  Maybe we should remove remark from test
+    Suite
```
Comment (maclover7) [inline meetings/2017-12-06.md]: Suite --> suite

File: meetings/2017-12-06.md [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+  * Refael has removed with objection.
+  * Jeremiah we should avoid pulling in many more files.  Suggestion is to remove markdown
+    linting until we can find another solution 
+  * James, has had enough problems that we should just remove markdown linting and then
+    fix afterwards.
+  * Myles ok with closing, need volunteer to remove linter.  Will close and @refack will take point
+    on removing/finding an alternative solution.  Make strategic initiative to around dependency
```
Comment (maclover7) [inline meetings/2017-12-06.md]: ✂️ `to`?

File: meetings/2017-12-06.md [hunk 6]
```diff
@@ -0,0 +1,163 @@
...
+  * Group of companies who have adopted Node.  
+  * Organizing group to provide feedback
+  * Goal is to have that group run the process, that is why no other TSC members organized
+  * Feedback on priorities, pain points.
+  * Michael, how do we position versus existing user-feedback effort to put call out to companies
+    who would participate in regular interlocks.
+  * James, talk to Dan on that front.
+  * Myles, what is being done to set expectations in terms of what actions may/may not happen
+    based on input.
+  * WIlliam, important that we set expectations. user-feedback group recognizes that setting
```
Comment (maclover7) [inline meetings/2017-12-06.md]: William

File: meetings/2017-12-06.md (hunk 1 above)
Comment (mhdawson) [reply meetings/2017-12-06.md]: I was meant to be underscore property, I'll just put that into words

**Comment by mhdawson** [comment]
@maclover7 thanks for the comments, updated based on comments.

**Comment by mhdawson** [comment]
Landed as 51d44a98bbf28ec839393c943351732fdb1a7bb4

**Pull request closed without merging**